client/src/panels/panel_declarations/index.js

```diff
@@ -21,9 +21,8 @@ export {
   declare_in_year_stat_breakdown_panel,
   declare_in_year_voted_stat_split_panel,
   declare_vote_stat_split_panel,
-  declare_welcome_mat_panel,
-} from //declare_tp_by_region_panel,
-"./finances/index.js";
+  declare_welcome_mat_panel, //declare_tp_by_region_panel,
+} from "./finances/index.js";
 
```

Comment:
Hmm, this and some other changes look like weird Prettier artifacts. Any idea what's happened? Could be from back when I rebased this branch...

Reply:
I think I may have changed this, as I was rebasing as it gave me conflicts. I'm not 100% sure as I went through like 60 different files to rebase this lol

Reply:
Ah, ouch, ok!